2001-3000/LIT2361somade.xml

```diff
@@ -30,13 +30,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
             
                <sourceDesc>
                   <listWit>
-                     <witness  corresp="BAVet87">V1</witness>             
-                     <witness  corresp="BAV134">V4</witness>
-                     <witness corresp="BLorient584">L1</witness>
-                     <witness corresp="BLadd16196">L6</witness>
-                     <witness  corresp="BNFabb106">P2</witness> 
-                     <witness  corresp="BNFet192">P3</witness>
-                     <witness  corresp="BNFet371">P5</witness>
+                     <witness  corresp="BAVet87"/>        
+                     <witness  corresp="BAV134"/>
```

Comment:
BAVet134 ?

Reply:
Yes, thank you!

Reply:
@karljonaskarlsson May I ask you to check Ms BAVet134, please? I do not see a record for BAVet134 anymore. Is it because you have not merged it yet or because we lost both records through the conflict?

Reply:
In the master branch, I can find the file that I created in "et" within the folder VaticanBAV.

However, when I try to open it in the app, I get the following message: 




<img width="539" alt="Skärmavbild 2024-09-24 kl  16 28 28" src="https://github.com/user-attachments/assets/c54b1532-7e1b-43b9-b7a9-3ee80da97590">



I think we might need the help of @eu-genia to resolve this